app/views/early_years_payment/practitioner/claims/how_we_use_your_information.html.erb

```diff
@@ -1,52 +1,123 @@
-<% title = "How we’ll use the information you provide" %>
+<% title = "How we'll process your claim" %>
```

Comment:
smart quotes + was this not the one you reverted?

Reply:
Sorted the smart quotes.
Yeah this was the one I reverted as it hadn't been business approved